bots/issue-scan

```diff
@@ -52,7 +52,7 @@ import argparse
 import pipes
 import os
 import sys
-import urllib
+import urllib.request, urllib.parse, urllib.error
```

Comment:
We don't enforce pycodestyle/PEP-8 in cockpit, but usually there should just be one import on every line.

Reply:
Fixed.